README.md

```diff
@@ -1,6 +1,6 @@
 # CLDF dataset accompanying List's "Inference of Partial Colexifications" from 2023 building on Key and Comrie's "Intercontinental Dictionary Series" from 2021
 
-[![CLDF validation](https://github.com/intercontinental-dictionary-series/idssegmented/workflows/CLDF-validation/badge.svg)](https://github.com/intercontinental-dictionary-series/idssegmented/actions?query=workflow%3ACLDF-validation)
+[![CLDF validation](https://github.com/intercontinental-dictionary-series/idssegmented//workflows/CLDF-validation/badge.svg)](https://github.com/intercontinental-dictionary-series/idssegmented//actions?query=workflow%3ACLDF-validation)
```

Comment:
Curious, there's a double `//` in the URL. Is there an issue with paths ending with `/`?

Reply:
not sure where this komes from actually. But it seems like a tag that is not added for other datasets. And it works 🤷